include/cpuinfo.h

```diff
@@ -1682,6 +1747,11 @@ uint32_t CPUINFO_ABI cpuinfo_get_l2_caches_count(void);
 uint32_t CPUINFO_ABI cpuinfo_get_l3_caches_count(void);
 uint32_t CPUINFO_ABI cpuinfo_get_l4_caches_count(void);
 
+/**
+ * Returns upper bound on cache size.
+ */
+uint32_t CPUINFO_ABI cpuinfo_get_max_cache_size(void);
```

Comment:
@Maratyszcza This function is not defined as far as I can tell.

Reply:
Looks like I missed it when creating a patch. The implementation is trivial: just return ‘cpuinfo_max_cache_size’